src/certificate/Certificate.test.tsx

```diff
@@ -140,6 +183,7 @@ function mockClient(loc: PublicLoc, item?: CollectionItem): LogionClient {
                 }
             },
             getTokensRecords: () => Promise.resolve([]),
```

Comment:
Should be aligned with below mock (`getTokensRecord`)?

Reply:
Yes, indeed. In this case this won't change anything because getTokensRecords() is called only when authenticated in token-gated mode, and this case is not covered (yet) by a test.